students/hayoung-gam/routes/UserRouter.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { UserController } from '../controllers';
+
+const router = express.Router();
+
+router.get('/', UserController.findUsersList);
```

Comment:
```suggestion
router.get('', UserController.findUsersList);
```
RESTful URL 시간에 알려드리겠지만, 미리 말씀드리자면 api url이 끝날 때에는 슬래시를 붙이지 않는 규칙이 있습니다 :)
(물론 회사마다 다르지만 보편적인 규칙을 알려드립니당)

Reply:
앗 감사합니다!
module layering을 재수정하며 이 부분도 함께 수정했습니다.

`/users` : https://github.com/wecode-bootcamp-korea/fullstack-1-westarbucks-backend-team1/pull/5/commits/fe05cf9dceb6af74d2c9549259b7f6c4b0683609

`/drinks` : https://github.com/wecode-bootcamp-korea/fullstack-1-westarbucks-backend-team1/pull/5/commits/b4ef5ab28f2d317c3e1e340fc650511ee2a656f7